hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/client/clustering/run/strategy/MultipleSparkJobExecutionStrategy.java

```diff
@@ -385,7 +385,7 @@ private HoodieData<HoodieRecord<T>> readRecordsForGroupBaseFiles(JavaSparkContex
 
   private HoodieFileReader getBaseOrBootstrapFileReader(StorageConfiguration<?> storageConf, String bootstrapBasePath, Option<String[]> partitionFields, ClusteringOperation clusteringOp)
       throws IOException {
-    HoodieFileReader baseFileReader = HoodieFileReaderFactory.getReaderFactory(recordType)
+    HoodieFileReader baseFileReader = (new HoodieSparkIOFactory()).getReaderFactory(recordType)
```

Comment:
Create a util to return a static `HoodieSparkIOFactory` instance for Spark engine?